### PR TITLE
fix: removed templates and image references

### PR DESCRIPTION
**Pull request by i-chvets**
# Description
Details are in https://github.com/canonical/katib-operators/issues/132

Summary of changes:
- Updated config and default custom images JSON to exclude image configuration options.
- Removed 2 templates.
- Updated default trial template to reference Ubuntu image.
- Modified integration tests to handle update template.

### CODE REVIEW

**Review by orfeas-k — COMMENTED** [review]
Thanks for the work @i-chvets, left some comments.

File: charms/katib-controller/src/templates/defaultTrialTemplate.yaml.j2 [hunk 1]
```diff
@@ -8,7 +8,7 @@ spec:
           image: {{ default_trial_template }}
           command:
             - "python3"
-            - "/opt/mxnet-mnist/mnist.py"
+            - "/opt/some-script.py"
```
[the remark this answers is not in the document]
Comment (i-chvets) [reply charms/katib-controller/src/templates/defaultTrialTemplate.yaml.j2:11]: The above change is showing how command is created with some sample script. We don't want to put anything spefic.

File: charms/katib-controller/src/default-custom-images.json [hunk 2]
```diff
@@ -1,12 +1,9 @@
 {
-    "default_trial_template": "docker.io/kubeflowkatib/mxnet-mnist:v0.16.0-rc.1",
+    "default_trial_template": "docker.io/ubuntu:22.04",
     "early_stopping__medianstop": "docker.io/kubeflowkatib/earlystopping-medianstop:v0.16.0-rc.1",
-    "enas_cpu_template": "docker.io/kubeflowkatib/enas-cnn-cifar10-cpu:v0.16.0-rc.1",
```
[the remark this answers is not in the document]
Comment (i-chvets) [reply charms/katib-controller/src/default-custom-images.json:4]: +1

**Review by orfeas-k — APPROVED** [review]
Thank @i-chvets

**Pull request merged**